### PR TITLE
Add unit tests for isLocked Library

### PR DESCRIPTION
**Pull request by VenkateshPS**
Add unit tests for isLocked Library
https://github.com/appium/python-client/issues/283

### CODE REVIEW

File: test/unit/webdriver/device/islocked_test.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    @httpretty.activate
+    def test_islocked_false(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/is_locked'),
+            body='{"value": false}'
+        )
+        assert driver.is_locked() == False
```
Comment (KazuCocoa) [inline test/unit/webdriver/device/islocked_test.py]: What about `assert xxxx is False` syntax

Comment (VenkateshPS) [reply test/unit/webdriver/device/islocked_test.py]: What about assert xxxx is False syntax ?

Comment (KazuCocoa) [reply test/unit/webdriver/device/islocked_test.py]: https://stackoverflow.com/questions/27276610/boolean-identity-true-vs-is-true
`is`, `is not`

Comment (VenkateshPS) [reply test/unit/webdriver/device/islocked_test.py]: Thanks done

**Pull request merged**